app/selfserve/module/Olcs/src/Controller/Lva/AbstractUploadEvidenceController.php

```diff
@@ -87,6 +87,7 @@ public function indexAction()
                     \Dvsa\Olcs\Transfer\Command\Application\UploadEvidence::create($dtoData)
                 );
                 if ($result->isOk()) {
+                    $this->addSuccessMessage('Your document upload was successful');
```

Comment:
Of course @Ahmed7752  the jira ticket doesn't mention this so not your fault... but it's required by legislation that we translate messages on selfserve into Welsh :)

Not sure you've ever been told/shown the process but can talk you through it tomorrow if needed.